pkg/csi/manila/identityserver.go

```diff
@@ -30,10 +31,16 @@ type identityServer struct {
 }
 
 func (ids *identityServer) GetPluginInfo(ctx context.Context, req *csi.GetPluginInfoRequest) (*csi.GetPluginInfoResponse, error) {
+	klog.V(5).Infof("Using default GetPluginInfo")
+
 	if ids.d.name == "" {
 		return nil, status.Error(codes.Unavailable, "Driver name not configured")
 	}
 
+	if ids.d.fqVersion == "" {
+		return nil, status.Error(codes.Unavailable, "Driver is missing version")
+	}
```

Comment:
What happened previously when this was not set? Did it fail, or did it just log incorrectly? If the latter you'll want to at least add a release note for this.

Reply:
Assuming I am reading things correctly, it would have failed since the field is required:

https://github.com/container-storage-interface/spec/blob/98819c45a37a67e0cd466bd02b813faf91af4e45/lib/go/csi/csi.pb.go#L693-L694